bin/setup

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash 
 set -e
 
+# Fixes UTF-8 error message on some machines.
```

Comment:
@macornwell Nit: The UTF-8 thing is a warning, not an error. For a workaround like this it can be good to be explicit. How about:

```
# Fixes UTF-8 warning ("Your terminal is not configured to receive UTF-8 encoded text.") on some machines.
```